package.json

```diff
@@ -25,6 +25,7 @@
     "babel-plugin-transform-object-rest-spread": "^6.6.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-1": "^6.5.0",
```

Comment:
What are you using from stage-1 that wasn't included individually? I'd prefer to do individual features so we can keep track of what is used I think.